src/app/home-page/new-dashboard/new-dashboard.component.html

```diff
@@ -8,7 +8,7 @@
       <app-entry-box class="fx-grow box-container" entryType="TOOL"></app-entry-box>
       <app-entry-box class="fx-grow box-container" entryType="SERVICE"></app-entry-box>
     </div>
-    <div id="updates-holder" fxLayout="row" fxLayoutGap="2.5rem" class="w-100 fx-grow">
+    <div id="updates-holder" fxLayout="row" fxLayoutGap="2.5rem" class="w-100 fx-grow updates-holder">
       <mat-card class="fx-grow">
```

Comment:
I recommend using `fxFlex` here instead of the `fx-grow` styling class because it removes the need for that class